### PR TITLE
Fix installation when a version is pinned and the installer is enabled

### PR DESCRIPTION
**Pull request by chouquette**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -453,3 +469,6 @@ workflows:
          matrix:
            parameters:
               ansible_version: ["2_10", "3_4", "4_10"]
+
+      - test_installer_over_pinned
```
Comment (Pythyu) [inline .circleci/config.yml:473]: 🐒 Nitpick :
Looks like a sed error

File: tasks/main.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 - name: Debian Install Tasks
   include_tasks: pkg-debian.yml
-  when: ansible_facts.os_family == "Debian" and not agent_datadog_skip_install
```
Comment (Pythyu) [inline tasks/main.yml:38]: ❓ Question :
Are we still using `agent_datadog_skip_install` or it's unsupported from this PR on ?

**Review by Pythyu — APPROVED** [review]
Otherwise LGTM 👍

File: tasks/main.yml (hunk 2 above)
Comment (chouquette) [reply tasks/main.yml:38]: It's still read from down the line, and is used by the installer to disable installing the agent through the package manager

File: .circleci/config.yml (hunk 1 above)
Comment (chouquette) [reply .circleci/config.yml:473]: It's adding the job defined above to the workflow :grin: 

I don't think there's a way to define and add the job as a single step, or at least I haven't found one :(

Comment (Pythyu) [reply .circleci/config.yml:473]: Aaah I see didn't knew we could call it like that 👍

**Pull request merged**